DataStandardization/create_data_to_process_files.py

```diff
@@ -5,19 +5,37 @@
 
 RelevantTypes = ()
 
+def path_to_list(path):
+  folders = []
+  while True:
+    path, folder = os.path.split(path)
+    if folder:
+      folders.append(folder)
+    else:
+      if path:
+        folders.append(path)
+      break
+  folders.reverse()
+  return folders
+
+def sep_maker():
+  list = ['a','b']
+  x = os.path.join(*list)
+  return x[1]
+
 
 def checkfunction(dtype):
     # write_dir(output_directory, INPUT_DATA, "PFI", list_of_dTypes, os.listdir(d_type_directory), Covariate_dir)
-    if not os.path.exists(os.path.dirname(f'{output_directory}/{dtype}.txt')):
-        File1 = open(f'{output_directory}/{dtype}.txt', 'w+')
+    if not os.path.exists(os.path.dirname(f'{output_directory}{_}{dtype}.txt')):
```

Comment:
DRY principle--since we have the exact same string 2x in a row, it's probably better to make a variable.

Reply:
Also, I'd probably use `delim` or `delimiter` instead of `_` as a variable name. From what I understand, `_` is usually used if you are trying to capture and throw away a return value.

Reply:
Oh, also, you can probably cut out this `if` statement and put `open(path, 'a+')`. [Link explaining that](https://stackoverflow.com/a/13248062/6481442).